vllm/entrypoints/openai/serving_chat.py

```diff
@@ -103,7 +102,7 @@ def _parse_chat_message_content_parts(
         parts: Iterable[ChatCompletionContentPartParam],
     ) -> ChatMessageParseResult:
         texts: List[str] = []
-        image_futures: List[Awaitable[ImagePixelData]] = []
```

Comment:
For consistency with offline inference, perhaps we should pass `{"image": <PIL.Image.Image object>}` instead of `ImageData` directly.

Reply:
yep, I changed to external compatible forms.